src/Tabs.tsx

```diff
@@ -1 +1,43 @@
-// implement a component here
+import { FC, useState } from 'react';
+
+interface TabsProps {
+  tabs: Tab[],
+}
+
+export const Tabs: FC<TabsProps> = ({ tabs }) => {
+  const [selectedTab, setSelectedTab] = useState('tab-1');
+  const [selectedContent, setSelectedContent] = useState('Some text 1');
+
```

Comment:
it works in this case, but if the title/content of the tab would change? It would be better to import the tabs array and select the first element, that way you could also just have one hook for the entire object.

Reply:
Okay. Thanx will make changes still fixing my PC